libmodulemd.spec.in

```diff
@@ -120,7 +131,10 @@ export MMD_SKIP_VALGRIND=1
 # Don't conflict with modulemd-validator from 1.x included in the official
 # RHEL 7 repos
 mv %{buildroot}%{_bindir}/modulemd-validator \
-   %{buildroot}%{_bindir}/modulemd-validator2
+   %{buildroot}%{_bindir}/modulemd-validator%{?v2_suffix}
+
+mv %{buildroot}%{_mandir}/man1/modulemd-validator.1* \
+   %{buildroot}%{_mandir}/man1/modulemd-validator%{?v2_suffix}.1*
```

Comment:
I don't think this is going to work. With the wildcard for the destination, you'll end up with the manpage named (literally) `modulemd-validator2.1*`. The `rename` command will do the right thing. (See my earlier feedback.)

Reply:
It did just that. From https://copr-be.cloud.fedoraproject.org/results/packit/fedora-modularity-libmodulemd-457/epel-7-x86_64/01240240-libmodulemd/builder-live.log.gz:
```
+ mv /builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg147db7f.el7.x86_64/usr/share/man/man1/modulemd-validator.1 '/builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg147db7f.el7.x86_64/usr/share/man/man1/modulemd-validator2.1*'
```

Reply:
That's actually a typo. I copied it from the `%files` section. It's guaranteed not to have the suffix at this point in the process, so I can just drop the asterisk.

Reply:
I thought the point of the wildcard when packaging manpages was to allow for them to have an optional compression suffix?

Reply:
It is, but the compression is handled "magically" by RPM itself between the `%install` and `%files` steps.

Reply:
Ah. OK. I know I saw somewhere that `meson` reserved the right to compress man pages when it recognized one during install. But now that I checked, it stopped doing that as of release 0.49.

https://mesonbuild.com/Release-notes-for-0-49-0.html#manpages-are-no-longer-compressed-implicitly

Reply:
Oh, hmm... that might indeed be an issue on epel-7... Let's see what happens with the Packit build.

Reply:
https://copr-be.cloud.fedoraproject.org/results/packit/fedora-modularity-libmodulemd-457/epel-7-x86_64/01240252-libmodulemd/builder-live.log shows:

```
+ mv /builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg269af30.el7.x86_64/usr/bin/modulemd-validator /builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg269af30.el7.x86_64/usr/bin/modulemd-validator2
+ mv /builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg269af30.el7.x86_64/usr/share/man/man1/modulemd-validator.1 /builddir/build/BUILDROOT/libmodulemd2-2.9.1-0.20200214.2gitg269af30.el7.x86_64/usr/share/man/man1/modulemd-validator2.1
```
So I think we're good.